### ISSUE TITLE
Tiles download

### ISSUE THREAD

**Issue by luigifab** [der-stefan/OpenTopoMap#309]
Currently, your are proposing to download maps for Garmin and Basecamp here: https://garmin.opentopomap.org
It is possible for you to propose also PNG tiles in multiple ZIP files (by keeping directories, eg: [/7/63/44.png](https://tile.opentopomap.org/7/63/44.png)) to allow us to use OpenTopoMap offline?

**Comment by luigifab** [comment]
Is there a python program that can read the previous img files?

**Comment by der-stefan** [comment]
No, bulk downloads of raster tiles is not planned. To be precise, the web server is throttling after a certain amount of downloaded tiles. Please understand that mass downloads would quickly overload the server.
